dali/python/nvidia/dali/pipeline.py

```diff
@@ -879,41 +879,43 @@ def _feed_input(self, name, data, layout=None, cuda_stream=None, use_copy_kernel
             self._pipe.SetExternalTLInput(name, data, ctypes.c_void_p(cuda_stream), use_copy_kernel)
 
     def feed_input(self, data_node, data, layout=None, cuda_stream=None, use_copy_kernel=False):
-        """Pass a multidimensional array or DLPack (or a list thereof) to an output of
-           ExternalSource.
+        """Pass a multidimensional array or DLPack (or a list thereof) to the eligible Operator.
```

Comment:
```suggestion
        """Pass a multidimensional array or DLPack (or a list thereof) to the eligible operator.
```

It is just a nitpick, but for me the capitialized operator reads weird here (like a term from the first section of super formal agreement where the definiton of parties is introduced).

I think input operator instead of InputOperators would be easier to read. For example you write `The operators that`, but in the same sentence `are the InputOperators`. Wouldn't simply operators, operator and input operators be enough? When referring to actual code you use `fn.inputs` and `fn.external_source`, which should be enough.

Reply:
Fixed